data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1226,6 +1226,9 @@ spec:
                   ovirt_cluster_id:
                     description: The target cluster under which all VMs will run
                     type: string
+                  ovirt_master_clone:
+                    description: MasterClone indicates that the disk of the master node should be cloned (thick provisioning)
```

Comment:
what's the reason for naming it this way? shouldn't it say something more tangible..like "use_disk_thick_provisioning" or "use_preallocated_disks" or something like that?

Reply:
maybe still keep "master" but eventually i would want this option to apply to workers too, even if the code for that is elsewhere. 
not necessarily now

Reply:
The oVirt API calls it clone and I believed it best to stay in convention. Adding an additional name for the same thing would increase potential confusion.

Reply:
on a second thought seeing the right comments throughout the code it's fine as is for me too, if it's better to align the parameter name with go-ovirt

Reply:
yeah. it's an unfortunate naming. this parameter is confusing everyone in oVirt to this day:/